src/CodeExplorerBundle/Twig/SourceCodeExtension.php

```diff
@@ -49,7 +49,7 @@ public function getFunctions()
 
     public function showSourceCode(\Twig_Environment $twig, \Twig_Template $template)
     {
-        return $twig->render('default/_source_code.html.twig', array(
+        return $twig->render('CodeExplorerBundle::source_code.html.twig', array(
```

Comment:
@javiereguiluz,  I think we should use it here. This notation more clearly for understanding. I like it.